### DISCUSSION TITLE
Falta Header en el API geodds

### DISCUSSION THREAD

**Discussion by PabloSorrentino** [dds-jv/foro#372] [Consultas de UI / REST] [answered]
Buenas, intentando probar la [API obligatoria](https://app.swaggerhub.com/apis-docs/ezequieloscarescobar/geodds/1.0.0#/default/get_api_distancia)  para manejar ubicaciones para la entrega 2° del TPA , al intentar hacer el request con esta URL "https://ddstpa.com.ar/api/distancia?localidadOrigenId=2&calleOrigen=maipu&alturaOrigen=100&localidadDestinoId=457&calleDestino=O%27Higgins&alturaDestino=200" o aun intentando desde la API misma con el boton TRYOUT nos tira el siguiente error : 

![image](https://user-images.githubusercontent.com/77159762/170804138-de15a07f-a124-4ccb-823b-d8e7082cb66b.png)

¿El error estaría de nuestro lado? 
¿Cómo lo solucionamos? 
Saludos.

**Comment by mbeorlegui** [comment, marked as answer]
Fijate de autorizar la API desde abajo de todo, donde te pide un ingreso de mail y te genera un token que tenes que usar para las requests.
